src/plugins/discover/public/build_services.ts

```diff
@@ -72,7 +72,7 @@ export interface DiscoverServices {
   docLinks: DocLinksStart;
   embeddable: EmbeddableStart;
   history: () => History<HistoryLocationState>;
-  theme: ChartsPluginStart['theme'];
```

Comment:
Since we migrated to Lens histogram, having charts theme in Kibana context is not necessary, right?